tests/test_example_dags.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+import pytest
+from airflow.executors.debug_executor import DebugExecutor
+from airflow.models.dagbag import DagBag
+
+# from astro.sql.operators.temp_hooks import PostgresHook
+from airflow.utils import timezone
+from airflow.utils.session import provide_session
+from airflow.utils.state import State
+from google.api_core.exceptions import NotFound
+
+# Import Operator
+
+DEFAULT_SCHEMA = "tmp_astro"
+DEFAULT_DATE = timezone.datetime(2016, 1, 1)
+
+
+@provide_session
+def get_session(session=None):
+    return session
+
+
+@pytest.fixture()
+def session():
+    return get_session()
+
+
+@pytest.mark.parametrize(
+    "dag_id",
+    [
+        "astro_test_dag",
+        "sql_file_dag",
+        "demo_with_s3_and_csv",
+        "snowflake_animal_adoption_example",
+    ],
+)
+def test_example_dag(session, dag_id):
```

Comment:
We were planning to run this in a periodic manner like a once a day or so, but this looks like part of CI?

Reply:
@utkarsharma2 yes this runs fairly quickly. I'm gonna attempt to split up the CI soon which should speed up everything considerably.